File: 프로그래머스/Level 1/행렬의 덧셈.cpp
//https://programmers.co.kr/learn/courses/30/lessons/12950

#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
    vector<vector<int>> answer;
    int size1 = arr1.size();
    int size2 = arr1[0].size();
    for(int i = 0; i < size1; i++){
        for(int j = 0; j < size2; j++){
            arr2[i][j] = arr2[i][j] + arr1[i][j];
        }
    }
    answer = arr2;
    return answer;
}

File: 프로그래머스/Level 1/행렬의 덧셈.py
//https://programmers.co.kr/learn/courses/30/lessons/12950?language=python3

import numpy as np
def solution(arr1, arr2):
    answer = []
    row = len(arr1)
    col = len(arr1[0])
    
    for i in range(row):
        tmp = []
        for j in range(col):
            tmp.append(arr1[i][j] + arr2[i][j])
        answer.append(tmp)
    return answer
